deployment/terraform/create.go

```diff
@@ -82,11 +85,17 @@ func New(id string, cfg deployment.Config) (*Terraform, error) {
 		return nil, fmt.Errorf("unable to get generated values: %w", err)
 	}
 
-	return &Terraform{
+	t := &Terraform{
 		id:        id,
 		config:    &cfg,
 		genValues: genValues,
-	}, nil
+	}
+
+	if err := t.InitCreds(); err != nil {
```

Comment:
Does this need to run outside of CI? If so, I'd be nice to have a `os.Getenv("CI") == true` or something.

Reply:
Good point! We do need to call `InitCreds` because that's how we authenticate now, but we can avoid the refresh goroutine if we're not in CI. Done in https://github.com/mattermost/mattermost-load-test-ng/pull/940/commits/5297daa8784db29d113791fb7d2d33fa769c6f3b.